openssl/src/pkcs7.rs

```diff
@@ -146,6 +183,82 @@ impl Pkcs7 {
             .map(Pkcs7)
         }
     }
+
+    /// Set signed attributes in a PKCS#7 structure.
+    ///
+    /// `attributes` is a stack of the attributes to be added.
+    ///
+    #[corresponds(PKCS7_set_signed_attributes)]
+    pub fn set_signed_attributes(
```

Comment:
Shouldn't this be a method on `Pkcs7SignerInfoRef`?

Reply:
Yes, that's better. Fixed.